includes/admin/views/builder/sidebar.php

```diff
@@ -2,15 +2,16 @@
 /**
  * Builder sidebar view template
  *
- * @since   3.16.0
- * @version 3.16.7
+ * @since 3.16.0
+ * @version [version]
```

Comment:
```suggestion
 * @since 3.16.0
 * @since [version] Added video explainer wrapper element.
 * @version [version]
```
In this case a changelog here would be useful, as we have no other places where to add it. What do you think?
Feel free to change the wording of course, dunno if you've heard but... I'm not a native english speaker 😆